app/views/gobierto_participation/processes/_group_teaser.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="pure-u-1 pure-u-md-1-3">
+  <%= link_to gobierto_participation_process_path(group.slug), class:"content_block light" do %>
+    <h3><%= group.title %></h3>
+    <p><%= group.body %></p>
+    <% if meta %>
+      <div class="meta right">
```

Comment:
¿Can this be implemented? If not, we use a `<% pending do %> ... <% end %>` helper in the views to hide content temporarily.